src/tests/data/test_context_search_test_context_search_succeeds.jsonl

```diff
@@ -1 +0,0 @@
-{"input": [{"text": "Role Description: The programmer receives requirements about a program and write it\\nTask: We need to render a highly complex 3D image on the solar system. We can use any publicly avaliable\\n    resources to achieve this task.\\n\\nRelevant Documents:\\n\\n\\n\\nReview and analyze the provided documents with respect to the task. Craft a concise and short, unified summary that distills key information that is most relevant to the task, incorporating reference links within the summary.\\nOnly use the information given. Do not add any additional information. The summary should be less than 5 setences\\n", "agent": "human"}], "output": "The task is to render a highly complex 3D image of the solar system. The programmer is required to use publicly available resources to achieve this task. No specific documents are provided for review and analysis.", "llm_name": "gpt-3.5-turbo"}
```

Comment:
Why is this file removed?

Reply:
i don't remember deleting it. will add it back